stl/inc/algorithm

```diff
@@ -2348,6 +2348,61 @@ _FwdIt2 replace_copy_if(_ExPo&&, _FwdIt1 _First, _FwdIt1 _Last, _FwdIt2 _Dest, _
 }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+
+    // VARIABLE ranges::fill
+    class _Fill_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        // clang-format off

```

Comment:
Is clang-format suppression necessary, given that there are no `requires` clauses here? Occurs below.

(General rule is to leave clang-format enabled unless the results look bad.)